dashboards/Experimental/MongoDB_ReplSet_Summary_(New).json

```diff
@@ -3147,38 +4069,11 @@
         "type": "query",
         "useTags": false
       },
-      {
-        "allFormat": "glob",
-        "current": {
-          "isNone": true,
-          "selected": false,
-          "text": "None",
-          "value": ""
-        },
-        "definition": "label_values(up{service_name=~\"$service_name\"},service_name)",
-        "hide": 2,
-        "includeAll": false,
-        "label": "",
-        "multi": false,
-        "name": "pod",
-        "options": [],
-        "query": {
-          "query": "label_values(up{service_name=~\"$service_name\"},service_name)",
-          "refId": "StandardVariableQuery"
-        },
-        "refresh": 2,
-        "regex": "/default-(.*-.*)/",
-        "skipUrlSync": false,
-        "sort": 0,
-        "tagsQuery": "",
-        "type": "query",
-        "useTags": false
-      },
       {
         "current": {
           "selected": false,
-          "text": "mongodb-27020",
-          "value": "mongodb-27020"
+          "text": "test-mongodb-shard00svr1.c.tams-207217.internal-mongodb",
```

Comment:
Is this correct?

Reply:
Hi this is dynamic, changes according to dashboards. have removed it.